src/component/organism/list/AssignCheckList.js

```diff
@@ -2,8 +2,7 @@ import React from 'react';
 import {FlatList} from 'react-native';
 import {View} from 'react-native';
 import AssignCheckListItem from 'Organism/listitem/AssignCheckListItem';
-import {assignCheckList} from 'Organism/style_organism';
-
+import {assignCheckList} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {assignCheckList} from 'Organism/style_organism copy';
```